var/spack/repos/builtin/packages/amdlibflame/package.py

```diff
@@ -69,21 +70,32 @@ def lapack_libs(self):
     def configure_args(self):
         """configure_args function"""
         args = super(Amdlibflame, self).configure_args()
-        args.append("--enable-external-lapack-interfaces")
 
-        """To enabled Fortran to C calling convention for
-        complex types when compiling with aocc flang"""
-        if "@3.0: %aocc" in self.spec:
-            args.append("--enable-f2c-dotc")
+        # 3.2.rc2 version onwards amd optimized flags are encapsulated under:
+        # enable-amd-flags for gcc compiler
+        # enable-amd-aocc-flags for aocc compiler
+        if "@3.2.rc2:" in self.spec:
+            if "%gcc" in self.spec:
+                args.append("--enable-amd-flags")
+            if "%aocc" in self.spec:
+                args.append("--enable-amd-aocc-flags")
```

Comment:
I don't think this portion of the code will apply to any version:
```console
Spack version 0.18.0
Python 3.8.10, Linux x86_64
>>> import spack.version
>>> v1 = spack.version.ver('3.2')
>>> v2 = spack.version.ver('3.2.rc2')
>>> v1 < v2
True
```

Reply:
Thanks for notifying the issue. The issue is now corrected.

This was meant for internal test purposes. 
My bad, we missed updating the amdlibflame recipe for the final release version.